src/styles/utils/variable.scss

```diff
@@ -0,0 +1,27 @@
+// Nav
+$nav-hight: 60px;
+$nav-color: #00acdc;
+$nav-link-color: #060b35;
+
+// Card-grid
+$column-count: 1;
+$small-min-width: 488px;
+$medium-min-width: 768px;
+$large-min-width: 1024px;
+
+// Card-size
+$main-card-width: 100px;
+$percent-width-img: 0.6;
+$percent-hight-img: 0.34;
```

Comment:
The variable name `$percent-hight-img` seems to have a typo. Consider renaming it to `$percent-height-img` to reflect the correct spelling.